### PR TITLE
Add the basic harness implementation

### PR DESCRIPTION
**Pull request by glesica**
Here's the thing. Let's talk about it.

### CODE REVIEW

File: harness/data/haystack.fa [hunk 1]
```diff
@@ -0,0 +1 @@
+atgtgatgccatgcacttacaaagaactatccatcagagctaccgaattacgcggagttacgttcggggtcttaatagcatttgaggaaatgtctgcaga
```
Comment (taylorreiter) [inline harness/data/haystack.fa]:
```suggestion
>my_fasta_1
atgtgatgccatgcacttacaaagaactatccatcagagctaccgaattacgcggagttacgttcggggtcttaatagcatttgaggaaatgtctgcaga
```

switch format to proper fasta format

Comment (glesica) [reply harness/data/haystack.fa]: The mock search doesn't understand fasta, but we definitely want this to be a valid file of one sort or another (depending on which formats we decide to support)

Comment (taylorreiter) [reply harness/data/haystack.fa]: awesome, thanks!

File: harness/the_count/wrapper.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+if [ $# != 2 ]
+then
+  echo "usage: wrapper.sh <needle.fa> <haystack.fa>"
+  exit
+fi
+# todo: should this script build the image?
```
Comment (glesica) [inline harness/the_count/wrapper.sh]: Nope, it's going to be run from inside the container.

**Comment by traviswheeler** [comment]
I suggest that we change the required `psss-benchmark` command to something generic, since we want this to live on beyond the psss project.  How about `tested-tool` ?

I'd make the change myself, but am afraid that I might miss something

**Comment by glesica** [comment]
I already changed it @traviswheeler, is there a specific place I missed? There are two commands, 'prepare_data' and 'run_tool'.

**Comment by traviswheeler** [comment]
@glesica : sorry, I should've written that this was for the_count and mock_search

**Comment by traviswheeler** [comment]
@glesica : The transmark harness prints results to standard out. What are your thoughts on adjusting that to write to a file? 

Related: in the above commits, we added more MSAs in the MSA files, to induce (and observe) a fan out. Now, when complete, there are a ton of newlines being printed.  How hard to reduce the number of these newlines?

**Comment by glesica** [comment]
I'm not seeing that many newlines...

```
1.0	2.0	5840	6163	rmark1	TRAIN.23S_rRNA_IVP.12
1.0	2.0	8614	8937	rmark1	TRAIN.23S_rRNA_IVP.12
1.0	2.0	9661	9984	rmark1	TRAIN.23S_rRNA_IVP.12
1.0	2.0	9738	10061	rmark1	TRAIN.23S_rRNA_IVP.12



psss-team3-hard-annotation/harness on  harness-spike [!?] took 10s
➜
```

That's the bottom of the output. Looks like two newlines. Were you seeing this with Slurm? Maybe that runner introduces some extra stuff?

We can easily write those to a file, but for now it's just more convenient to see them so that we can test quickly as we iterate on it. Ultimately, we're going to be feeding that output into post-processing, so that will probably mean writing it out to a file.

**Pull request merged**